src/jit/emit.h

```diff
@@ -932,7 +932,7 @@ class   emitter
             }
             void            iiaSetInstrCount(int count)
             {
-                assert(abs(count < 10));
+                assert(count < 10);
```

Comment:
I presume that this change is motivated by another compiler/tool error. I am not totally familiar with this code, but my assumption is that it should be assert((count >=0 && (count < 10)). A quick scan didn't indicate any context in which it should be negative.